_posts/2019-11-21-november-meetup.md

```diff
@@ -0,0 +1,42 @@
+---
+title: CopenhagenJS November - Penni.io
+type: meetup
+location: Frederiksholms Kanal 4, København
+link: https://www.meetup.com/copenhagenjs/events/265924609/
+date: 2019-11-21T17:00:00
+duration: 3
+speakers:
+  - title: 'Typescript: Mapped & Conditional Types'
+    name: Pavel Kučera
+---
+
+# CopenhagenJS November - Penni.io
+
+Hello everyone 😄,
+It is time for the November edition of CopenhagenJS. This time we are hosted by Penni.io.
+
+Schedule:
```

Comment:
These should be headlines like in https://github.com/copenhagenjs/copenhagenjs.dk/blame/master/_posts/2019-10-24-october-meetup.md#L20